tests/e2e/metrics_test.go

```diff
@@ -50,7 +50,7 @@ func metricsTest(cx ctlCtx) {
 		{"/metrics", fmt.Sprintf(`etcd_server_version{server_version="%s"} 1`, version.Version)},
 		{"/metrics", fmt.Sprintf(`etcd_cluster_version{cluster_version="%s"} 1`, version.Cluster(version.Version))},
 		{"/metrics", fmt.Sprintf(`grpc_server_handled_total{grpc_code="Canceled",grpc_method="Watch",grpc_service="etcdserverpb.Watch",grpc_type="bidi_stream"} 6`)},
-		{"/health", `{"health":"true"}`},
+		{"/health", `{"health":"true","reason":""}`},
```

Comment:
Looks like you unintentionally backported a "reason" field. It's not a breaking change but also not a "code cleanup".

Reply:
changed it to `json:"-"`. Would that work?